packages/fuels/tests/contracts/auth_testing_abi/Forc.toml

```diff
@@ -5,3 +5,5 @@ license = "Apache-2.0"
 name = "auth_testing_abi"
 
 [dependencies]
+core = { git = "https://github.com/FuelLabs/sway", branch = "Voxelot/fuel-core-0.13" }
```

Comment:
this was needed to force forc to use the updated sway libs instead of the older versions on the latest release tag